adafruit_turtle.py

```diff
@@ -238,20 +242,20 @@ def __init__(self, display=None, scale=1):
         self.pencolor(Color.WHITE)
         self._bg_pic = None
         self._bg_pic_filename = ""
-        self._turtle_pic = None
-        self._turtle_odb = None
-        self._turtle_alt_sprite = None
+        self._turtle_pic: Any = None
+        self._turtle_odb: Any = None
+        self._turtle_alt_sprite: Any = None
         self._drawturtle()
-        self._stamps = {}
+        self._stamps: Dict[int, Any] = {}
         self._turtle_odb_use = 0
-        self._turtle_odb_file = None
-        self._odb_tilegrid = None
+        self._turtle_odb_file: Optional[str] = None
+        self._odb_tilegrid: Any = None
```

Comment:
I think some of these could be narrower than `Any`, but they're also hidden, so it may be worth just leaving them untyped for now.